big_tests/tests/push_integration_SUITE.erl

```diff
@@ -310,28 +335,73 @@ send_message_to_room(Sender, Room) ->
 
 
 muclight_msg_notify_on_apns(Config, EnableOpts) ->
-    escalus:story(
+    escalus:fresh_story(
         Config, [{alice, 1}, {bob, 1}, {kate, 1}],
         fun(Alice, Bob, _Kate) ->
-            {SenderJID, DeviceToken} =
-                muclight_conversation(Config, Alice, Bob, <<"apns">>, EnableOpts),
+            RoomJID = muc_light_helper:given_muc_light_room(fresh_room_name(), Alice, [{Bob, member}]),
+            timer:sleep(5000),
```

Comment:
5 seconds sleep? Why such a long sleep? Shouldn't the tests be kept as fast as possible? :'(

Reply:
Oh, that's a leftover fortunately :')